### ISSUE TITLE
lowest/highest scores index not updated for season 3

### ISSUE THREAD

**Issue by lil-lilith** [Ravenousdead/MythicScore#3]
var lowestScore/highestScore on lines 170,171,181,182 are using season2 index, need to be updated from 9391_[9,10] to 1000010_[9,10]

**Comment by lil-lilith** [comment]
PR here: https://github.com/Ravenousdead/MythicScore/pull/4

**Comment by tripnine** [comment]
@lil-lilith any way to get that pr merged into main other than just waiting on ravenousdead? would love to not have to compile every time i wana check the chart :)

**Comment by lil-lilith** [comment]
@CoachTino it seems there's no way to push through the PR without input from @Ravenousdead . There is one other user @Asgaroth that opened a PR with their own fixes and is hosting a working version here https://asgaroth.github.io/MythicScore/ we can use in the meantime

**Comment by Asgaroth** [comment]
Yeah, I dont want to take any credit for it. But I had to host it, otherwise I'd have to keep running a local server and it was a hassle. The original author may not be monitoring GitHub but as soon as they merge the changes we can continue using the original.

**Comment by I-AM-T3X** [comment]
@Asgaroth can you please update it for season 4 and redeploy it on your deployment

**Comment by tripnine** [comment]
@I-AM-T3X i've updated it here
https://tripnine.github.io/
to be clear i just used @Ravenousdead 's existing code!

**Comment by I-AM-T3X** [comment]
> @I-AM-T3X i've updated it here https://tripnine.github.io/ to be clear i just used @Ravenousdead 's existing code!

thank you @tripnine
